pages/docs/guides/validate.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Validate AsyncAPI Documents"
+description: In this guide, we'll learn multiple ways to validate AsyncAPI documents.
+weight: 120
+---
+
+# Introduction
+In this guide, we'll learn multiple ways to validate AsyncAPI documents. 
+
+When people talk about AsyncAPI validation, sometimes they mean completely different things: **validation of AsyncAPI documents** or **validation of messages in runtime** _(message schemas provided in the AsyncAPI document)_.  
+
+# Prequisites
```

Comment:
I recommend we do not do prerequisites in guides. Each section is about different ways of validating and have different requirements. It is a guide that explains different ways of doing things, so people can choose one or 2 or all, depends. So defining prerequisites can be hard and also misleading. For example maybe someone do not need governance validation, so why would they install spectral.

I think it is completely fine to remove it

Reply:
Ok, sure 👍🏽